transform/mattermost-analytics/models/intermediate/web_app/platform/int_mm_telemetry_prod_performance_events.sql

```diff
@@ -0,0 +1,13 @@
+{{
+    config({
+        "materialized": "incremental",
+        "cluster_by": ['event_date'],
+    })
+}}
+
+SELECT * FROM 
+{{ ref('stg_mm_telemetry_prod__performance_events') }}
+
+{% if is_incremental() %}
+    WHERE event_date >= (SELECT MAX(event_date) FROM {{ this }}) 
```

Comment:
The reason I used this, is because I want the table to be preformative, mainly in Looker. The cluster_key is crucial for this reason, I don't have a problem to use `received_at`, provided the cluster is on `event_date`. Across the board for most analyses we use `timestamp` or `original_timestamp`

Reply:
Are the intermediate model tables used in Looker directly or only the fact/dim tables in the `marts` folder?

Reply:
only tables from `marts` folder